filament/src/ShadowMapManager.cpp

```diff
@@ -154,10 +166,12 @@ void ShadowMapManager::render(FrameGraph& fg, FEngine& engine, FView& view,
                     if (view.hasVsm()) {
                         renderTarget.attachments = { { data.shadows, 0u, i }, { data.tempShadow } };
```

Comment:
rename `renderTarget` to `renderTargetDesc`  also rename `shadowTexture` to `shadowTextureDesc` (generally use the `fooDesc` convention for descriptors)